src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -726,7 +726,8 @@ void net_device_table_mgr::handle_timer_expired(void* user_data)
 void net_device_table_mgr::global_ring_wakeup()
 {
 	ndtm_logdbg("");
-	struct epoll_event ev;
+	epoll_event ev = {0, {0}};
```

Comment:
Should do `{0}` and not `{0, {0}}`. It is more generic and a much more common and acceptable way of doing this.